src/Catty/DataModel/Bricks/Motion/PointToBrick.m

```diff
@@ -84,6 +84,14 @@ - (id)mutableCopyWithContext:(CBMutableCopyContext*)context
     return copy;
 }
 
+- (id)mutableCopyWithContext:(CBMutableCopyContext*)context AndErrorReporting:(BOOL)reportError
```

Comment:
Can you give me some hints why you added this method?

Reply:
The override of method with two arguments (reportError Boolean) wasn't present. So calls made by also passing the boolean for reportError always went to the parent method. Hence, added this argument.